packages/indexer-agent/src/commands/start.ts

```diff
@@ -78,7 +78,16 @@ export default {
       .option('gas-price-max', {
         description: 'The maximum gas price (gwei) to use for transactions',
         type: 'number',
-        default: 50,
+        default: 100,
+        deprecated: true,
+        group: 'Ethereum',
+        coerce: arg => arg * 10 ** 9,
+      })
+      .option('base-fee-per-gas-max', {
```

Comment:
The way I have it now there are no require updates for indexers since the new `base-fee-per-gas-max` and the old `gas-price-max` can be used interchangeably, so their old `gas-price-max` configs will still work.   We can phase that out in a future release.